cmd/controller/disk.go

```diff
@@ -28,12 +28,17 @@ type DiskInfo struct {
 	Serial           string          // Serial is serial no of a disk
 	Vendor           string          // Vendor is vendor of a disk
 	Path             string          // Path is dev path of a disk like /dev/sda
+	FirmwareRevision string          // FirmwareRevision is the firmware revision for a disk
+	LogicalSize      uint32          // LogicalSize is the Logical size of a disk in bytes
+	Version          string          // Version indicates the implemented version of the
+	// standards/specifications such as SPC-1, SPC-2, etc
 }
 
-// ProbeIdentifiers contains some keys to uniquely identify each disk by probe
+// ProbeIdentifier contains some keys to uniquely identify each disk by probe
```

Comment:
Update the comment with the need to define the identifiers separately from the disk details. I am not clear on why we need to duplicate these values here and disk-details.